skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerMetrics.scala

```diff
@@ -115,50 +118,226 @@ object ConsumerMetrics {
+          .observe(bytes.toDouble)
+        a <- age.foldMapM { age =>
+          ageSummary
+            .labels(clientId, topic)
+            .observe(age.toNanos.nanosToSeconds)
         }
+      } yield a
+    }
 
-        def count(name: String, topic: Topic) = {
-          callsCounter
-            .labels(clientId, topic, name)
-            .inc()
-        }
+    override def count(name: String, topic: Topic): F[Unit] = {
+      callsCounter
+        .labels(clientId, topic, name)
+        .inc()
+    }
 
-        def rebalance(name: String, topicPartition: TopicPartition) = {
-          rebalancesCounter
-            .labels(clientId, topicPartition.topic, name)
-            .inc()
-        }
+    override def rebalance(name: String, topicPartition: TopicPartition): F[Unit] = {
+      rebalancesCounter
+        .labels(clientId, topicPartition.topic, name)
+        .inc()
+    }
 
-        def topics(latency: FiniteDuration) = {
-          topicsLatency
-            .labels(clientId)
-            .observe(latency.toNanos.nanosToSeconds)
+    override def topics(latency: FiniteDuration): F[Unit] = {
+      topicsLatency
+        .labels(clientId)
+        .observe(latency.toNanos.nanosToSeconds)
+    }
+  }
+
+  def histograms[F[_]: Monad](
+    registry: CollectorRegistry[F],
+    prefix: Prefix = Prefix.Default
+  ): Resource[F, ClientId => ConsumerMetrics[F]] = {
+
+    val callsCounter = registry.counter(
+      name   = s"${prefix}_calls",
+      help   = "Number of topic calls",
+      labels = LabelNames("client", "topic", "type")
+    )
+
+    val resultCounter = registry.counter(
+      name   = s"${prefix}_results",
+      help   = "Topic call result: success or failure",
+      labels = LabelNames("client", "topic", "type", "result")
+    )
+
+    val latencyHistogram = registry.histogram(
+      name    = s"${prefix}_latency",
+      help    = "Topic call latency in seconds",
+      buckets = latencyBuckets,
+      labels  = LabelNames("client", "topic", "type")
+    )
+
+    val recordsHistogram = registry.histogram(
+      name    = s"${prefix}_poll_records",
+      help    = "Number of records per poll",
+      buckets = pollCountBuckets,
+      labels  = LabelNames("client", "topic")
+    )
+
+    val bytesHistogram = registry.histogram(
+      name    = s"${prefix}_poll_bytes",
+      help    = "Number of bytes per poll",
+      buckets = pollBytesBuckets,
+      labels  = LabelNames("client", "topic")
+    )
+
+    val rebalancesCounter = registry.counter(
+      name   = s"${prefix}_rebalances",
+      help   = "Number of rebalances",
+      labels = LabelNames("client", "topic", "type")
+    )
+
+    val topicsLatency = registry.histogram(
+      name    = s"${prefix}_topics_latency",
+      help    = "List topics latency in seconds",
+      buckets = latencyBuckets,
+      labels  = LabelNames("client")
+    )
+    val ageHistogram = registry.histogram(
+      name    = s"${prefix}_poll_age",
```

Comment:
here and for other metrics, please apply naming rules as described on https://prometheus.io/docs/practices/naming/

```suggestion
      name    = s"${prefix}_poll_age_seconds",
```